flink-python/pyflink/table/tests/test_row_based_operation.py

```diff
@@ -125,9 +123,11 @@ def test_aggregate_with_pandas_udaf(self):
                                [DataTypes.FIELD("a", DataTypes.FLOAT()),
                                 DataTypes.FIELD("b", DataTypes.INT())]),
                            func_type="pandas")
-        t.group_by(t.a) \
-            .aggregate(pandas_udaf(t.b).alias("c", "d")) \
-            .select("a, c, d").execute_insert("Results") \
+        t.select(t.a, t.b) \
+            .group_by(t.a) \
+            .aggregate(pandas_udaf) \
+            .select("*") \
```

Comment:
I guess `.select("*")`  could be removed?

Reply:
`AggregateTable` need to call `select` method to convert to `Table`